src/screens/Main/EditStack/SnippetSelection/SnippetSelection.tsx

```diff
@@ -20,120 +20,115 @@ interface Props {
   navigation: NavigationScreenProp<NavigationState>,
 }
 
+const updateSnippetState = function(
```

Comment:
you are saving this function into variable but you don't need to. the syntax is simply
`function updateSnippetState(args) {}`